### ISSUE TITLE
Selling ship/equipment = bank/merchant buys back at reduced price

### ISSUE THREAD

**Issue by airtroop392** [endless-sky/endless-sky#312]
If you were to sell a ship/outfit, the bank/merchant would buy it at a fraction of what you originally paid for it (around 80%). This would encourage players to invest their money wisely, rather than spending it all over the place.

**Comment by gchapyzh** [comment]
I would vote no. I often take the chance that buy=sell and experiment with my ship outfits a lot. If this were to be implemented, there would also certainly have to be a way to test outfits without wasting money.

**Comment by Pointedstick** [comment]
I agree with gchapyzh. I also do the buy-and-test-out dance, and this would heavily penalize that play style.

The only place where I see this as a problem is the high price you can get for looted equipment, which is currently an easy ticket to wealth as soon as you have a mildly battle-worthy ship with a reasonable cargo bay.

**Comment by Oeleo** [comment]
Yes I agree too, selling outfit at a lower price but a way to test outfit.

**Comment by gchapyzh** [comment]
Let me rephrase that: Useless without a dynamic economy.

**Comment by airtroop392** [comment]
The fact that everyone in the galaxy will happily buy back your used equipment, but you get loans with 0.4% interest seems oddly out of place. I must admit that I do like the

**Comment by Oeleo** [comment]
Yes globally it needs to be harder to make money, that's why I agree with the idea of taxe, more you have things more it become difficult to make money. And that's why I agree with this idea too, furthermore because in 99% of the games stuff that isn't new is sell at a lower price, that's seem logical.

**Comment by Oeleo** [comment]
So they buy it to you 100% of the price and they sell it 100% ? But they do... 0% of benefit ?

**Comment by endless-sky** [comment]
As others have said, the point of this is to allow the player to test out different equipment loadouts without needing to worry that they are losing money every time they swap out an outfit, and without the risk that the player will be left in a state where they have sold their only engines or their only generator and then do not have enough money to buy it back again.

**Comment by airtroop392** [comment]
I'll drop the issue, then. The general consensus has been that there is no need, and the dev himself weighed in on this. Defend this idea as I may, I appear to be sorely outnumbered.
